### ISSUE TITLE
Out-of-distribution generation

### ISSUE THREAD

**Issue by CeMOS-Abdullah** [boschresearch/ALDM#7]
Is it possible to generate out-of-distribution samples using custom colormaps without the need to finetune(similar to the FreestyleNet implementation here: https://github.com/essunny310/FreestyleNet)? 

According to my understanding of the codebase, the inference seems to be limited to only the colormaps defined in the dataloader modules for the respective datasets (coco-stuff/ADE20K/Cityscapes).

If yes, could you provide an example, or outline the steps needed to implement this feature?

Thanks in advance!

**Comment by YumengLi007** [comment]
Hi @CeMOS-Abdullah ,
the input label map condition is constrained to the classes defined in the dataloaders. However, one can still change the prompt to generate out-of-distribution samples, e.g., different weather conditions, a burning van, etc. as shown in the paper.

**Comment by CeMOS-Abdullah** [comment]
Thank you very much for your reply.
